python/pyspark/sql/tests.py

```diff
@@ -1518,6 +1518,12 @@ def test_stream_trigger(self):
         except ValueError:
             pass
 
+        # Should not take multiple args
+        try:
```

Comment:
@tdas, could we maybe test this by `assertRaises` if it checks raising an exception?